client/plots/sampleScatter.js

```diff
@@ -130,7 +129,7 @@ class Scatter {
 		const s0 = this.data.samples[0]
```

Comment:
can you change to a meaningful variable name?

Reply:
s0 is the first sample, is visible from the value it gets and also from the name(short of sample0).  I added a comment to this line to explain it. Let me know if you want me to rename the variable though.